File: 백준/Bronze/17388. 와글와글 숭고한/와글와글 숭고한.cc
#include<stdio.h>

int main() {
    int S,K,H;
    scanf("%d %d %d",&S,&K,&H);
    if(S + K + H >= 100){
        printf("OK");
    }else{
        if(S < K && S < H){printf("Soongsil");}
        if(K < S && K < H){printf("Korea");}
        if(H < S && H < K){printf("Hanyang");}
    }
}

File: 백준/Bronze/17388. 와글와글 숭고한/와글와글 숭고한.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
    Scanner myObj = new Scanner(System.in); 
    
    int S = myObj.nextInt();
    int K = myObj.nextInt();
    int H = myObj.nextInt();
    String R = "Hello";
    if(S + K + H >= 100){
        R = ("OK");
    }else{
        if(S < K && S < H){R = ("Soongsil");}
        if(K < S && K < H){R = ("Korea");}
        if(H < S && H < K){R = ("Hanyang");}
    }
    System.out.println(R);
    }
}